tasks/src/test/java/webdriver/google_cloud/service/GoogleCloudService.java

```diff
@@ -1,10 +1,11 @@
 package webdriver.google_cloud.service;
 
-import static webdriver.google_cloud.constants.Constants.*;
+import static webdriver.google_cloud.util.StringUtils.*;
```

Comment:
Does the service class really belong to the StringUtils package?
The package name doesn't comply with Java package naming convention